.github/workflows/scheduled.yml

```
@@ -396,3 +403,46 @@ jobs:
           name: join-fuzzer-failure-artifacts
           path: |
             /tmp/join_fuzzer_repro
+
+  linux-exchange-fuzzer-run:
+    runs-on: ubuntu-latest
+    needs: compile
+    timeout-minutes: 120
+    steps:
+
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+        with:
+          ref: "${{ inputs.ref || 'main' }}"
+
+      - name: Install dependencies
+        run: sudo ./scripts/setup-ubuntu.sh
```

```suggestion
        run: source ./scripts/setup-ubuntu.sh
```
running the script is not necessary, we just need the apt depdendencies. This saves ~15 minutes